documentation/FAQ.md

```diff
@@ -25,7 +41,7 @@ Frequently Asked Questions
     $ sudo codesign --force --sign - /usr/local/bin/lldb
     ```
 
-This will make a copy of Xcode lldb and adhoc sign it. Note that this workaround is only applicable to Xcode lldb, not the System lldb (`/usr/bin/lldb`), which may still fail to load. You may need to delete or rename `~/.lldbinit` to use `/usr/bin/lldb`.
+    This will make a copy of Xcode lldb and adhoc sign it. Note that this workaround is only applicable to Xcode lldb, not the System lldb (`/usr/bin/lldb`), which may still fail to load. You may need to delete or rename `~/.lldbinit` to use `/usr/bin/lldb`.
```

Comment:
I'm not sure if adding this indent was intentional, but I assume the markdown formatting won't honor it.

Reply:
Yes, it was intentional and does affect the formatting (at least in the VS markdown extension I have).